tests/integration_tests/test_experiment.py

```diff
@@ -187,14 +187,14 @@ def test_experiment_multiclass_with_class_weights(csv_filename):
 def test_experiment_multilabel_with_class_weights(csv_filename):
     # Multiple inputs, Single category output
     input_features = [category_feature(vocab_size=10)]
-    output_features = [set_feature(vocab_size=3, loss={"class_weights": [0, 0, 1, 2, 3]})]
+    output_features = [set_feature(vocab_size=3, loss={"class_weights": [0, 1, 2, 3]})]
```

Comment:
Not 100% sure how padding is done now in set features, as removing that 0 means there's only unk, right?

Reply:
Right -- there's only UNK. As discussed offline, the padding symbol doesn't seem necessarily since we convert set and bag features to a vector of binarys/ints.